packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -0,0 +1,409 @@
+                                        });
+                                        onPasswordManager(
+                                            mt5_trade_account?.login,
+                                            getTitle(mt5_trade_account.market_type, is_eu_user),
+                                            mt5_trade_account.account_type,
+                                            account_type,
+                                            (mt5_trade_account as DetailsOfEachMT5Loginid)?.server
+                                        );
+                                        toggleModal();
+                                    }}
+                                />
+                            </div>
+                        </div>
+                    </>
+                )}
+
+                <div className='cfd-trade-modal__maintenance'>
+                    <Icon
+                        icon='IcAlertWarning'
+                        size={isMobile() ? 28 : 20}
+                        className='cfd-trade-modal__maintenance-icon'
+                    />
+                    <div className='cfd-trade-modal__maintenance-text'>
+                        <Localize i18n_default_text='Server maintenance starts at 06:00 GMT every Sunday and may last up to 2 hours. You may experience service disruption during this time.' />
+                    </div>
+                </div>
+            </div>
+            <div className='cfd-trade-modal__download-center-app'>{downloadCenterAppOption(platform)}</div>
+            <div className='cfd-trade-modal__download-center-options'>
+                <div className='cfd-trade-modal__download-center-options--mobile-links'>
+                    <div className='cfd-trade-modal__download-center-options--mobile-links--apple'>
+                        <a href={mobileDownloadLink(platform, 'ios')} target='_blank' rel='noopener noreferrer'>
+                            <Icon icon='IcInstallationApple' width={isMobile() ? '160' : '130'} height={40} />
+                        </a>
+                    </div>
+                    <a href={mobileDownloadLink(platform, 'android')} target='_blank' rel='noopener noreferrer'>
+                        <Icon icon='IcInstallationGoogle' width={135} height={40} />
+                    </a>
+                    {platform !== 'ctrader' && (
+                        <div className='cfd-trade-modal__download-center-options--mobile-links--huawei'>
+                            <a href={mobileDownloadLink(platform, 'huawei')} target='_blank' rel='noopener noreferrer'>
+                                <Icon icon='IcInstallationHuawei' width={135} height={40} />
+                            </a>
+                        </div>
+                    )}
+                </div>
+                <div className='cfd-trade-modal__download-center-options--qrcode'>{differentQRCodes(platform)}</div>
+            </div>
+        </div>
+    );
+};
+
+export default TradeModal;
```

Comment:
@hirad-deriv The file is too big, We can extract reusable parts into separate components and compose them together here 🙇🏻

Reply:
That's correct. Almost all parts of the component are not used anywhere else hence instead of putting different parts in the Components file I am putting different parts in a config file to make this component smaller and more maintainable.